src/smt/dump.cpp

```diff
@@ -248,12 +260,6 @@ t-explanations [non-stateful]\n\
 bv-rewrites [non-stateful]\n\
 + Output correctness queries for all bitvector rewrites\n\
 \n\
-bv-abstraction [non-stateful]\n\
```

Comment:
Should also remove documentation for the other removed ones, e.g. `state`, not sure if there are others.

Reply:
Discussed offline.